docs/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -450,10 +449,18 @@ As a workaround, create or modify `.npmrc` in your project root:
 ```npmrc
 shamefully-hoist=true
 ```
+
 <Alert level="warning">
   **Note**: While `shamefully-hoist=true` usually isn't the ideal solution from a dependency management perspective, it's sometimes necessary for compatibility with certain packages that expect Node.js module resolution behavior similar to npm or yarn.
 </Alert>
 
+{" "}
+<Alert level="warning">
```

Comment:
Looks like this note is getting repeated 👯
<img width="734" alt="Screenshot 2024-12-11 at 4 21 00 PM" src="https://github.com/user-attachments/assets/a4921d66-8114-4023-8f90-2cbeb59a8241" />